src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -38,4 +38,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CentralPackageVersionsFileImported>true</CentralPackageVersionsFileImported>
   </PropertyGroup>
 
+  <!-- Extension point after Directory.Packages.props is imported. -->
+  <Import Project="$(CustomAfterDirectoryPackagesProps)" Condition="'$(CentralPackageVersionsFileImported)' == 'true' and '$(CustomAfterDirectoryPackagesProps)' != ''" />
```

Comment:
Might be a good idea to add a test. 
Examples: https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

Reply:
I was hoping that we don't need a test for such a trivial change. I.e. we don't have tests for such extension points in the SDK / MSBuild Common targets. If you feel strongly about it, I'm fine with adding one.

Reply:
I think a unit test in this case would just make sure it was regressed in the future

Reply:
Do we have a unit test for the import in L35 above that I could use as a template?